tests/test_integrations.py

```diff
@@ -813,3 +819,36 @@ def test_loading_correctly(monkeypatch, interactive):
 def test_exec_function_scope(cmd):
     _, _, rtn = run_xonsh(cmd, single_command=True)
     assert rtn == 0
+
+
+@skip_if_on_unix
+def test_run_currentfolder():
+    """Ensure we can run an executable in the current folder
+    when file is not on path
+    """
+    batfile = Path(__file__).parent / "bin" / "hello_world.bat"
+    os.chdir(batfile.parent)
```

Comment:
Use the monkeypatch fixture with `monkeypatch.chdir`

Reply:
It's probably the reason other tests are failing

Reply:
Yes. Let me just try that. Thanks